src/ui/layout/main-block.svelte

```diff
@@ -29,4 +39,48 @@
     display: flex;
     background-color: var(--rc-color-background);
   }
+
+  .layout-wrapper-outer {
+    flex: 1;
+    display: flex;
+    background-color: var(--rc-color-background);
+  }
+
+  .layout-wrapper {
+    width: 100%;
+  }
+
+  .layout-content {
+    box-sizing: border-box;
+    background-color: var(--rc-color-background);
+    color: var(--rc-color-grey-text-dark);
+    display: flex;
+    flex-direction: column;
+    padding: var(--rc-spacing-outerPadding-mobile);
+  }
+
+  @container layout-query-container (width < 768px) {
+    .layout-wrapper {
+      width: 100%;
+      min-width: 300px;
+      display: flex;
+      flex-grow: 1;
+    }
+
+    .layout-content {
+      flex-grow: 1;
+      height: 100%;
+    }
+  }
+
+  @container layout-query-container (width >= 768px) {
+    .layout-wrapper {
+      min-height: 100vh;
```

Comment:
this is now duplicated in here and navbar, and makes it a bit more brittle no?

Reply:
Agreed! Let me see if I can work it out now that the back button is not there on mobile anymore.